cooperator_website_referral_source/views/referral_source_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="referral_source_action" model="ir.actions.act_window">
+        <field name="name">How did you hear about us?</field>
+        <field name="res_model">referral.source</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+
+    <menuitem
+        id="referral_source_menu"
+        name="Referral Source"
+        action="referral_source_action"
+        parent="cooperator.menu_cooperator_config"
+        sequence="999"
```

Comment:
Is that mandatory to write `sequence=999` ?

Reply:
Je l'ai ajouté car la demande est que le menu soit à la fin de la liste des menus de configuration.